frontend/ui/widget/opdsbrowser.lua

```diff
@@ -504,7 +504,7 @@ function OPDSBrowser:downloadFile(item, format, remote_url)
     logger.dbg("downloading file", local_path, "from", remote_url)
 
     local parsed = url.parse(remote_url)
-    http.TIMEOUT, https.TIMEOUT = 10, 10
+    http.TIMEOUT, https.TIMEOUT = 20, 20
     local httpRequest = parsed.scheme == 'http' and http.request or https.request
     local _, c, _ = httpRequest{
```

Comment:
Well yes, it is certainly unused. Using regular _ hides that fact either because it's used elsewhere in its gettext capacity or because _ is a common dummy variable. I suppose I should've added an @houqp. But it definitely seems a bit odd to say "hey, _ is this now, but let's immediately change it back to what it was."